study-builder/config/cmi-config.json.ctmpl

```diff
@@ -0,0 +1,51 @@
+{{with $environment := env "ENVIRONMENT"}}
+{{with $version := env "VERSION"}}
+{{with $response := vault (printf "secret/pepper/%s/%s/cmi/conf" $environment $version)}}
+{{with $conf := $response.Data}}
+
+{
+  "AUTH0_DOMAIN": "{{$conf.auth0.domain}}",
+  "AUTH0_CLIENT_ID": "{{$conf.auth0.clientId}}",
+  "AUTH0_CLIENT_SECRET": "{{$conf.auth0.clientSecret}}",
+  "AUTH0_KEYWORD_REPLACE_MAPPINGS": {
+  {{if eq $environment "dev"}}
+    {{/* These end up being a line item in a yaml file, so the hyphen in front is needed. */}}
+    "LOCALHOST_CALLBACKS": "- http://localhost:4200/login-landing",
+    "LOCALHOST_LOGOUTS": "- http://localhost:4200, http://localhost:4200/session-expired",
+    "LOCALHOST_ORIGINS": "- http://localhost:4200",
+  {{else}}
+    {{/* Need to set these as blank, otherwise they won't get substituted and auth0 deploy will complain. */}}
```

Comment:
Used consul template comments here since this ends up being a json file and there's no comments in json...